packages/page-claims/src/index.tsx

```diff
@@ -235,46 +235,52 @@ function ClaimsApp (): React.ReactElement {
             )}
           {(step >= Step.Sign && !isPreclaimed) && (
             <Card>
-              <h3>{t('{{step}}. Sign with you ETH address',
-                { replace: { step: isOldClaimProcess ? '2' : '3' } })}</h3>
-              <Statement
-                kind={statementKind}
-              />
-              <div>
-                {t('Copy the following string and sign it with the Ethreum account you used during the pre-sale in the wallet of your choice, using the string as the payload, and then paste the transaction signature object below')}
-                  :
-              </div>
-              <CopyToClipboard
-                onCopy={onCopy}
-                text={payload}
-              >
-                <Payload
-                  data-for='tx-payload'
-                  data-tip
-                >
-                  {payload}
-                </Payload>
-              </CopyToClipboard>
-              <Tooltip
-                place='right'
-                text={didCopy ? t('copied') : t('click to copy')}
-                trigger='tx-payload'
-              />
-              <Signature
-                onChange={onChangeSignature}
-                placeholder={`{\n  "address": "0x ...",\n  "msg": "${prefix}:...",\n  "sig": "0x ...",\n  "version": "2"\n}`}
-                rows={10}
-              />
-              {(step === Step.Sign) && (
-                <Button.Group>
-                  <Button
-                    icon='sign-in'
-                    isDisabled={!accountId || !signature}
-                    label={t('Confirm claim')}
-                    onClick={goToStepClaim}
+              {(isOldClaimProcess || (statementKind?.isAlternative || statementKind?.isDefault))
```

Comment:
```suggestion
              {(isOldClaimProcess || !!statementKind)
```

If i'm not mistaken, that should work too, because statementKind will be `null | undefined` in that case.

Reply:
yes this works, thanks